tests/unit/calendar/calendar.normal.spec.js

```diff
@@ -26,12 +25,8 @@ describe('Calendar (normal mode)', () => {
 
     it('should not mutate props', () => {
       const mods = ['test'];
-      const wrapper = shallow(
-        <Calendar
-          mods={mods}
-        />
-      );
-      expect(shallowToJson(wrapper)).toMatchSnapshot();
+
+      shallow(<Calendar mods={mods} />);
```

Comment:
According to the test, the idea is to just verify if the props are mutated. That does not require the shallowToJson